src/app/ui/palette_popup.cpp

```diff
@@ -42,6 +42,7 @@ PalettePopup::PalettePopup()
   addChild(m_popup);
 
   m_paletteListBox.DoubleClickItem.connect(base::Bind<void>(&PalettePopup::onLoadPal, this));
+  m_paletteListBox.Change.connect(base::Bind<void>(&PalettePopup::onSearchChange, this));
```

Comment:
Actually reviewing this line I checked that `ui::ListBox::Change` signal is generated when the selection is changed (on `ui::ListBox::onChange`), so this connection is a little confusing. I'll merge this PR as it is but adding a new signal when the resources are completely loaded so we can call `onSearchChange()` indirectly (instead of using the `ListBox::Change` signal).